src/QSymbolicsBase/basic_ops_homogeneous.jl

```diff
@@ -82,7 +87,73 @@ Base.show(io::IO, x::SAddOperator) = print(io, "("*join(map(string, arguments(x)
 const SAddBra = SAdd{AbstractBra}
 Base.show(io::IO, x::SAddBra) = print(io, "("*join(map(string, arguments(x)),"+")::String*")") # type assert to help inference
 
-"""Tensor product of quantum objects (kets, operators, or bras)."""
+# defines commutativity for addition of quantum objects
+const CommQObj = Union{SAddBra, SAddKet, SAddOperator} 
```

Comment:
What is this for? It does not seem to be used anywhere else.

Reply:
During my tests, commutativity for addition did not always hold, so I added an `isequal` method for adding operations. Take the following example:
```
julia> A = SOperator(:A, SpinBasis(1//2)); B = SOperator(:B, SpinBasis(1//2));

julia> dagger(A+B)
(A†+B†)

julia> dagger(A+B)
(B†+A†)
```
For reasons that I'm still trying to determine, `dagger(A+B)` flips between the ordering of the operators when it simplifies. This also happens with the commutators when you expand them. I could spend time working on ordering objects in expressions (as is done in `SymbolicUtils`) to make sure this doesn't happen. That being said, before I defined `isequal` for adding operations, the test would break because `(A†+B†)` was not equal to `(B†+A†)`, which obviously isn't what we want. So, to at least make sure addition of objects are commutative, I added an additional `isequal` method.